h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2319,6 +2325,9 @@ protected void updateProgress(boolean canScore){
       if(canScore && (_parms._score_each_iteration || timeSinceLastScoring() > _scoringInterval)) {
         _model.update(_state.expandBeta(_state.beta()), -1, -1, _state._iter);
         scoreAndUpdateModel();
+        _earlyStop = !_earlyStop?ScoreKeeper.stopEarly(_model.scoreKeepers(),
```

Comment:
readability of this could be improved - probably extract the expression to method since the ternary expression seems to me too complicated already

Reply:
I simplified it.